supplier_stock/models/supplier_stock.py

```diff
@@ -117,11 +123,13 @@ def _onchange_partner_loc_id(self):
             self.currency_id = self.partner_loc_id.currency_id
 
     @api.multi
+    @api.depends('price_unit')
```

Comment:
Since the method depends on the `currency_id` as well
Let's make it ` @api.depends('price_unit', 'currency_id')`

Reply:
Fixed.